### DISCUSSION TITLE
Itron SCM+ ERT RF Registers 1, 2, and 3

### DISCUSSION THREAD

**Discussion by tgbanshee** [bemasher/rtlamr#255] [Q&A]
I am currently performing testing with RTLAMR using Itron Centron polyphase CP3S and single-phase CN1SR electric meters. I have noticed that the SCM+ message decoded by RTLAMR is only showing register 1 in the output. Register 2 & 3 are not outputted/decoded. There is also additional diagnostic/error data which looks to be transmitted but not decoded as well.

After reviewing the code, it looks like decoding of register 2 & 3 may not be currently supported. Is this the case or did I miss something and RTLAMR needs to be run with additional command-line switches or environment variables to decode register 2 & 3?

See below for screenshots of the RF section of the Itron meter configuration software (PC-PRO+ Advanced v9.85.21). The test program shown is specific to the polyphase CP3S electric meter.

![PC-Pro Screenshot 1](https://user-images.githubusercontent.com/444623/173612140-207a11dd-c51c-4bc3-9445-54add2361a80.jpg)

![PC-Pro Screenshot 2](https://user-images.githubusercontent.com/444623/173612161-d83cd543-cc36-4633-9dfa-5e13bdd3b3f1.jpg)

**Comment by vtjnash** [comment]
I think I see Endpoint id, Endpoint id+1, Endpoint id+2 always transmitting SCM+ in rapid sequence, so I'm guessing those all come from my meter (C1SDR3) for each register. I don't have a PC-PRO, so I don't know what each of those are configured to transmit however. Units appear to be kWh, Wh, and unknown and the values appear to hold constant from the previous meter reading from my bill last month.
